ci-operator/step-registry/cucushift/agent/vsphere/conf/cucushift-agent-vsphere-conf-commands.sh

```diff
@@ -164,8 +158,34 @@ for ((i = 0; i < total_host; i++)); do
     cidr=25
   else
     ipaddress=$(jq -r --argjson N $((i + 4)) --arg PRH "$primaryrouterhostname" --arg VLANID "$vlanid" '.[$PRH][$VLANID].ipAddresses[$N]' "${SUBNETS_CONFIG}")
+    ipv6_address=$(jq -r --arg PRH "$primaryrouterhostname" --arg VLANID "$vlanid" '.[$PRH][$VLANID].StartIPv6Address' "${SUBNETS_CONFIG}")
   fi
-
+  ipv4="
+        ipv4:
+          enabled: true
+          address:
+            - ip: ${ipaddress}
+              prefix-length: ${cidr}
+          dhcp: false"
+  ipv6="
+        ipv6:
+          enabled: true
+          address:
+            - ip: ${ipv6_address}
+              prefix-length: ${cidr_ipv6}
+          dhcp: false"
+  route_ipv4="
+          - destination: 0.0.0.0/0
+            next-hop-address: ${gateway}
+            next-hop-interface: ens32
+            table-id: 254"
+  route_ipv6="
+          - destination: ::/0
+            next-hop-address: ${gateway_ipv6}
+            next-hop-interface: ens32
+            table-id: 254"
+  if [[ ${IP_FAMILIES} == "IPv4" ]]; then ipv6=""; route_ipv6=""; fi
+  if [[ ${IP_FAMILIES} == "IPv6" ]]; then ipv4=""; route_ipv4=""; fi
```

Comment:
@mhanss, clearing the ipv4/ipv6 variables when the `IP_FAMILIES` is the contrary is working as expected for dual_stack? shouldn't be better to set them on a condition basis instead of clearing them after?

Reply:
If it's a dual-stack scenario, both conditions will be bypassed, and the values for IPv4 and IPv6 will be configured. Setting these values based on conditions would complicate the process and increase the number of lines in the code.

Reply:
@mhanss, ok, I think I see the logic now, those conditions only apply to single-stack options, right?

If that's the case, can you please at least put some comment above those lines to clarify? what about something like this?

~~~
  # Single-stack override conditions
  if [[ ${IP_FAMILIES} == "IPv4" ]]; then ipv6=""; route_ipv6=""; fi
  if [[ ${IP_FAMILIES} == "IPv6" ]]; then ipv4=""; route_ipv4=""; fi
~~~

Reply:
@pamoedom understood. The changes have been applied.